content/doc/book/platform-information/upgrade-java-to-17.adoc

```diff
@@ -0,0 +1,50 @@
+---
+layout: subsection
+title: Upgrading to Java 17
+---
+
+There are a few details and steps to upgrade the JVM used to run Jenkins, more specifically from Java 11 to Java 17.
+
+If you're upgrading the JVM used to run Jenkins, particularly from Java 11 to Java 17, there are some details you should know and precautions you should take.
+
+.Upgrading Jenkins Java Version From 11 to 17 
+video::ZabUz6sl-8I[youtube,width=800,height=420]
+
+== Backing up Jenkins
+
+As with any upgrade, link:/doc/book/system-administration/backing-up/#jenkins_home[we recommend backing up] `JENKINS_HOME` and testing the upgrade with the backup before performing the upgrade on your production instance.
```

Comment:
As with any upgrade, we recommend:
 
. Backing up `JENKINS_HOME`.
. Testing the upgrade with your backup.
. Only after all required tests pass, performing the upgrade on your production instance.

Reply:
I would include the link to backing up `JENKINS_HOME` in the steps below.

Reply:
I would suggest using a numbered list to make the recommended steps very clear.